docs/eventing/channels/default-channels-crds.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Default channels"
```

Comment:
There is a defaults-channels.md which should be marked as deprecated in 0.8, and maybe point to this one? What is the procedure to properly do that?

Reply:
@nachocano  We don't really have a mechanism for redirects right now, as far as I know, but I'll check with Richie when he gets back from vacation. I think what we'll do is delete the deprecated file from the 0.8 release; it'll live on in the <0.7 versions.

Reply:
@RichieEscarez